### PR TITLE
Create derived UserAccountViewModel for User (non-Organization) accounts

### PR DESCRIPTION
**Pull request by chenriksson**
Preparation for #4990.

Refactoring to create base AccountViewModel, to be used by both UserAccountViewModel and OrganizationAccountViewModel (coming w/ Manage Organization).

### CODE REVIEW

File: src/NuGetGallery/Controllers/UsersController.cs [hunk 1]
```diff
@@ -22,6 +22,21 @@ namespace NuGetGallery
     public partial class UsersController
         : AppController
     {
+        public static class Actions
```
Comment (scottbommarito) [inline src/NuGetGallery/Controllers/UsersController.cs]: I like this change but
1) It is necessary for this PR? I can't see a huge reason to do it right now, and it clutters up the actual changes you're making here.
2) Could we use these constants in `Route.cs` too?

Comment (chenriksson) [reply src/NuGetGallery/Controllers/UsersController.cs]: Completely true, I just got sick of seeing so many duplicated strings. :-)

If I want to keep this, will split into separate PR.

Comment (scottbommarito) [reply src/NuGetGallery/Controllers/UsersController.cs]: Feel free to go ahead and keep it--it isn't too much of a change. I was just wondering if there was a huge reason to do it.

Comment (chenriksson) [reply src/NuGetGallery/Controllers/UsersController.cs]: Separate PR makes sense, especially if I want to extend it to Routes.cs like you suggest.

**Pull request merged**